### PR TITLE
Jb45571

### PR DESCRIPTION
**Pull request by martyone**

### CODE REVIEW

**Comment by martyone** [comment]
Just the last two commits belong to this PR

File: src/libs/sfdk/emulator.cpp [hunk 1]
```diff
@@ -377,9 +377,12 @@ bool EmulatorPrivate::fromMap(const QVariantMap &data)
 
     DeviceModelData deviceModel;
     EmulatorManager::fromMap(&deviceModel, data.value(Constants::EMULATOR_DEVICE_MODEL).toMap());
-    setDisplayProperties(deviceModel,
-            static_cast<Qt::Orientation>(data.value(Constants::EMULATOR_ORIENTATION).toInt()),
-            data.value(Constants::EMULATOR_VIEW_SCALED).toBool());
+    Qt::Orientation orientation = static_cast<Qt::Orientation>(
+            data.value(Constants::EMULATOR_ORIENTATION, Qt::Vertical).toInt());
+    if (orientation == 0)
+        orientation = Qt::Vertical;
```
Comment (vigejolla) [inline src/libs/sfdk/emulator.cpp:383]: Given that there are only two allowed values, I'd write this as:
`Qt::Orientation orientation = data.value(Constants::EMULATOR_ORIENTATION, Qt::Vertical).toInt() == Qt::Horizontal = Qt::Horizontal ? Qt::Vertical`

Comment (martyone) [reply src/libs/sfdk/emulator.cpp:383]: The `if ...` part is there only to fix user configuration broken by this bug in past. It is meant to be removed after a few releases. Hence the separation.

Comment (vigejolla) [reply src/libs/sfdk/emulator.cpp:383]: Ok, in that case it makes sense.

**Review by vigejolla — APPROVED** [review]
LGTM

**Pull request merged**